pigeon/notifications.dart

```diff
@@ -77,4 +116,9 @@ abstract class AndroidNotificationHostApi {
     // NotificationCompat.Builder has lots more methods; add as needed.
     // Keep them alphabetized, for easy comparison with that class's docs.
   });
+
+  /// Corresponds to `androidx.core.app.NotificationManagerCompat.createNotificationChannel`.
+  ///
+  /// See: https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#createNotificationChannel(androidx.core.app.NotificationChannelCompat)
+  void createNotificationChannel(NotificationChannel channel);
```

Comment:
I am not sure if this could return `Future<void>` instead of `void` in here too, as the autogenerated concrete class has a return type of `Future<void>`. Looking at the docs from the link provided, it seems that this method has a return type of `void`, but that's in Java, and it doesn't have a type `Future`; at least that's how I know so far about Java. 😀

Reply:
This file is a Pigeon file, not a standard Dart file. It contains types and methods that define the native plugin API interface. See [Pigeon's example](https://github.com/flutter/packages/blob/main/packages/pigeon/example/README.md#dart-input).

Adding a `Future<void>` here will result in an error:
```
Error: pigeon/notifications.dart:120: Unknown type: Future
```

Reply:
Ok thanks, got it!